webserver/cors/src/main/java/io/helidon/webserver/cors/CorsSupportHelper.java

```diff
@@ -510,26 +524,16 @@ R processCorsPreFlightRequest(CrossOriginConfig crossOrigin,
                     () -> "actual origin: " + originOpt.get() + ", allowedOrigins: " + allowedOrigins);
         }
 
-        Optional<String> methodOpt = requestAdapter.firstHeader(ACCESS_CONTROL_REQUEST_METHOD);
-        if (methodOpt.isEmpty()) {
-            return forbid(requestAdapter,
-                    responseAdapter,
-                    METHOD_NOT_IN_ALLOWED_LIST,
-                    () -> "header " + ACCESS_CONTROL_REQUEST_METHOD + " absent from request");
-        }
-
         // Check if method is allowed
-        String method = methodOpt.get();
         List<String> allowedMethods = Arrays.asList(crossOrigin.allowMethods());
         if (!allowedMethods.contains("*")
-                && !contains(method, allowedMethods, String::equals)) {
+                && !contains(requestedMethod, allowedMethods, String::equals)) {
```

Comment:
HTTP methods should be compared ignore case (unless you fix the requestedMethod and configured methods somewhere to be always upper/lower cased)